sale_order_report_product_image/i18n/sale_order_report_product_image.pot

```diff
@@ -0,0 +1,19 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* sale_order_report_product_image
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```

Comment:
"Project-Id-Version: Odoo Server 15.0\n" shall be update ?

Reply:
Yes but translations are not manage in the github repo but by an external tool.

Reply:
Okey, thanks for the explanation :+1: